zipline/algorithm.py

```diff
@@ -1253,6 +1254,8 @@ def future_chain(self, root_symbol, as_of_date=None):
             Date at which the chain determination is rooted. I.e. the
             existing contract whose notice date is first after this date is
             the primary contract, etc.
+        offset: int
+            Number of sessions to go ahead or behind the given as_of_date.
```

Comment:
The prepositional phrases here don't quite work out.  It would need to be "ahead **of** or behind".  We also might want to explicitly spell out how negative and positive offsets work.

Suggestion: "Number of sessions to shift ``as_of_date``. Positive values shift forward in time. Negative values shift backward in time."  We could also add an `Examples` section below.